sound/soc/sof/topology.c

```diff
@@ -2132,9 +2142,20 @@ static int sof_widget_unload(struct snd_soc_component *scomp,
 		break;
 	case snd_soc_dapm_scheduler:
 		mutex_lock(&sdev->cores_mask_mutex);
+		pipeline = swidget->private;
 
+		/* someone is still using the core, return */
```

Comment:
@keyonjie sorry for the nit again. "someone" sounds odd. Can we just say "return if core is still in use" instead?

Reply:
good, let me update it.